src/main/java/frc/robot/subsystems/tower/ArmImpl.java

```diff
@@ -33,19 +28,24 @@
 
     //Arm PID
     private final int kPIDLoopIdx;
-    private final Gains kGains;
+    private final frc.robot.subsystems.Gains kGains;
```

Comment:
Re-add the import for Gains rather than referencing the fully-qualified package name here